_test/test_transformation/test_projection_class.m

```diff
@@ -82,6 +82,203 @@ function test_cut(this)
             %w = w{1};
             %wc = cut_dnd(w,0.01,0.01,[-0.1,0.1],2);
         end
+        function test_uv_to_rlu_and_vv_complex_nonorth_with_rrr(~)
+            u = [1,1,0];
+            v = [0,-0.5,1];
+            alatt = [2.83,2,3.83];
+            angdeg = [95,85,97];
+            pra = projaxes(u,v,'type','rrr');
+            %pra.nonorthogonal = true;

```

Comment:
Any reason for this to be here and or commented?

Reply:
because I was not able to implement this option. if this is possible or not at all -- the subject of discussion with Toby

Reply:
The  comment left as reminder of not implemented. may be it worth additional comment to justify this